test/images/image-util.sh

```diff
@@ -91,6 +91,11 @@ build() {
       make -C "${image}" bin OS="${os_name}" ARCH="${arch}" TARGET="${temp_dir}"
     fi
     pushd "${temp_dir}"
+
+    # we might have to build multiple images with the same name, but different versions.
+    # in this case, they will have different folders. in order to keep the same name, they will
+    # have an ALIAS file with the actual image name.
+    image_name=$(cat ALIAS 2> /dev/null || echo "${image}")
```

Comment:
Sure, that makes sense. Though, I renamed ``image_name`` to ``alias_name`` in order to avoid future confusion as to what is what.

Also, I opted to go with this version (echo-ing a message for logging purposes if an alias is found), which means that I can't create a function for it without complicating it a bit (echos would be captured by assignments: ``image=$(getImageName "$image")``

Reply:
I had to update it to ``alias_name="$(cat ALIAS 2>/dev/null || true)"``. If there was no ALIAS file, it wouldn't have failed entirely.